### DISCUSSION TITLE
Redirect to GitHub Discussions breaks Google search results

### DISCUSSION THREAD

**Discussion by tobloef** [ImageMagick/ImageMagick#5247] [General] [answered]
When searching for a question related to ImageMagick, quite often you will get results from https://legacy.imagemagick.org/discourse-server/. But it seems that posts on this site have recently begun to automatically redirect to https://github.com/ImageMagick/ImageMagick/discussions.

See https://legacy.imagemagick.org/discourse-server/viewtopic.php?t=35018 for an example.

This change makes it very hard to find answers that in the past have helped me tremendously. How can we access these old posts? And why was this redirect set up? Would you possibly please reconsider the change, as all those forum posts and the knowledge they contained are now rendered inaccessible (at least through google)?

**Comment by urban-warrior** [comment, marked as answer]
We restored the legacy discussions @ https://imagemagick.org/discourse-server.   It may take awhile to update the search indexes for the legacy topics.
